### PR TITLE
Skip unknown packages

### PR DESCRIPTION
**Pull request by SkySkimmer**
I don't know if set -e was a great idea, it keeps burning us in annoying ways.

### CODE REVIEW

File: two_points_on_the_same_branch.sh [hunk 1]
```diff
@@ -248,6 +248,9 @@ for coq_opam_package in $coq_opam_packages; do
         opam list
         echo "DEBUG: coq_opam_package = $coq_opam_package"
         opam show $coq_opam_package || continue 2
+    else
+        # cause to skip with error if unknown package
+        opam show $coq_opam_package >/dev/null || continue 2
```
Comment (ejgallego) [inline two_points_on_the_same_branch.sh:253]: Maybe you'd like to add some message error "Unknown package foo"?

Why is CI failing? [Coq bignums fail to intall]

I am not expert on bash, so we could remove -e if you want, tho I am more comfortable handling errors explicitly.

Comment (SkySkimmer) [reply two_points_on_the_same_branch.sh:253]: >Maybe you'd like to add some message error "Unknown package foo"?

opam will say that on stderr

Comment (SkySkimmer) [reply two_points_on_the_same_branch.sh:253]: >Why is CI failing? [Coq bignums fail to intall]

No clue, we should move to gitlab so we can upload logs as artifacts.

Comment (ejgallego) [reply two_points_on_the_same_branch.sh:253]: I'm all for it; but I won't have time to do that soon, maybe we could build with `-v` if `BENCH_DEBUG` is set?

**Comment by ejgallego** [comment]
Well I'm merging this anyways, I don't this the bignums failure is related.

**Comment by ejgallego** [comment]
Oh, I understand now; we are pinning a Coq commit in the CI, I'd try to fix that.

**Review by ejgallego — APPROVED** [review]

**Pull request merged**